src/vmm/src/persist.rs

```diff
@@ -235,8 +241,7 @@ fn snapshot_state_to_file(
     snapshot
         .save(&mut snapshot_file, microvm_state)
         .map_err(SerializeMicrovmState)?;
-
-    Ok(())
+    snapshot_file.sync_all().map_err(SnapshotFileFlush)
```

Comment:
No `file.flush()` here like in the block implementation?

I understand it's a noop. Regardless of the reason for including it anyway, should we also have it here as well?